app/views/jst/developer_key.handlebars

```diff
@@ -52,9 +52,11 @@
         </div>
```

Comment:
"Never" (on the line above this, can't comment there for some reason) needs to be `{{#t}}Never{{/t}}`